src/config/module-map.ts

```diff
@@ -4,4 +4,6 @@
  *  contents.
  */
 
-export default moduleMap = {};
```

Comment:
@locks `moduleMap` needs to be declared. I get `Cannot find name 'moduleMap'.` in VSCode (which pains me to admit being a Vim user).